src/components/custom-field-input-date/custom-field-input-date.test.jsx

```diff
@@ -254,4 +255,16 @@ describe('src/components/custom-field-input-date/custom-field-input-date', () =>
       expect(inputRef.current.value).toBe('2016-07-18');
     });
   });
+
+  describe('API limits', () => {
+    xit('shows the expected error text when a date is out of range', () => {
```

Comment:
Is this still supposed to be skipped?

Reply:
So we are having trouble getting userEvent typing to fill out the date field via typing in a date. Due to the nature of the date and multi / single select components, testing them has given us some problems. If you look up in the file there is another suspended spec that you all wrote that maybe due to a similar problem? userEvent.type seems to not be updating the text field. Would love your help in figuring this out, and then we can un pend both specs in this file.

Reply:
Oh right, we also had trouble with `type="date"`. I think we need to open up an issue in `@testing-library/user-event`. 

In the mean time, I think we got test coverage by using the calendar pop-up to change the dates. 

@bennapp should be able to add a bit more detail.